README.md

```diff
@@ -1,12 +1,16 @@
-# ActiveRecord::Bitemporal[ ![](https://img.shields.io/gem/v/activerecord-bitemporal.svg)](https://rubygems.org/gems/activerecord-bitemporal) [ ![](https://img.shields.io/gem/dt/activerecord-bitemporal.svg)](https://rubygems.org/gems/activerecord-bitemporal) [![CircleCI](https://circleci.com/gh/kufu/activerecord-bitemporal.svg?style=svg)](https://circleci.com/gh/kufu/activerecord-bitemporal)
+ActiveRecord::Bitemporal
+========================
+
+[![License](https://img.shields.io/github/license/kufu/activerecord-bitemporal.svg?color=blue)](https://github.com/kufu/activerecord-bitemporal/blob/master/LICENSE)
+[![gem-version](https://img.shields.io/gem/v/activerecord-bitemporal.svg)](https://rubygems.org/gems/activerecord-bitemporal)
+[![gem-download](https://img.shields.io/gem/dt/activerecord-bitemporal.svg)](https://rubygems.org/gems/activerecord-bitemporal)
+[![CircleCI](https://circleci.com/gh/kufu/activerecord-bitemporal.svg?style=svg)](https://circleci.com/gh/kufu/activerecord-bitemporal)
 
 ## Installation
 
 Add this line to your application's Gemfile:
 
-```ruby
-gem 'activerecord-bitemporal'
-```
+    $ gem 'activerecord-bitemporal'
```

Comment:
@paveg 
I think original one is better since this line is for Gemfile, not for shell.
Would you please undo this change?

Reply:
I misunderstood🤦‍♂️
I'll change it.